lms/static/lms/js/build.js

```diff
@@ -34,7 +34,8 @@
             'support/js/certificates_factory',
             'support/js/enrollment_factory',
             'js/bookmarks/bookmarks_factory',
-            'js/learner_dashboard/program_list_factory'
+            'js/learner_dashboard/program_list_factory',
+            'js/vendor/draggabilly',
```

Comment:
Draggabilly should not be listed here. This list is for high-level pages, where we want all dependencies of those pages to be bundled into a single file.